File: src/26. Remove Duplicates from Sorted Array/26.c
int removeDuplicates(int* nums, int numsSize){
    if (numsSize < 1) {
        return numsSize;
    }
    int j = 1;
    for (int i = 1; i < numsSize; i++) {
        if (nums[i] != nums[i-1]) {
            nums[j] = nums[i];
            j++;
        }
    }
    return j;
}

File: src/26. Remove Duplicates from Sorted Array/26.go
func removeDuplicates(nums []int) int {
    if len(nums) <= 1 {
        return len(nums)
    }
    j := 1
    for i:=1; i < len(nums); i++ {
        if nums[i] != nums[i-1] {
            nums[j] = nums[i]
            j++
        }
    }
    return j
}